runtimes/cpu/cpu.zig

```diff
@@ -13,10 +19,25 @@ pub fn load() !*const pjrt.Api {
         return error.Unavailable;
     }
 
+    var arena = std.heap.ArenaAllocator.init(std.heap.c_allocator);
```

Comment:
I'd rather avoid std.heap.c_allocator. `std.heap.smp_allocator` is just as fast and doesn't depend on libc.